processes/security_team_onboarding.md

```diff
@@ -0,0 +1,39 @@
+
+## Introducing new members
+New members may be nominated to join the WG, where they share an affinity to web or software security research, or activism around security and the Node.js project. All of which make good criteria to join the team but are not a closed list.
+
+### The Buddy Program
+Every new member should be assigned a *buddy* to help mentor and coach them through the on-boarding period, relevant processes and activities that the WG handles. 
+
+A *buddy* is an active member of the WG who is familiar with the above topics.
+
+A suggested on-boarding session for triage team members: 30 minutes touch-base call to cover the triage process on HackerOne and the repo.
+
+
+### How to nominate new members?
+Open an issue in https://github.com/nodejs/security-wg that suggests to nominate a new team member. Provide as much background and context as possible which is relevant to this candidate and his potential future contributions to the Security WG.
+
+References:
+* [Nominate Marcin Hoppe as member of the WG · Issue #163 · nodejs/security-wg · GitHub](https://github.com/nodejs/security-wg/issues/163)
+* [Nominate Gergely Nemeth as WG member by vdeturckheim · Pull Request #128 · nodejs/security-wg · GitHub](https://github.com/nodejs/security-wg/pull/128)
+
+
+### What happens next?
+Patiently wait for feedback and +1’s from the team and a reasonable time to ensure nobody disagrees (7 days).
+
+## Setting-up for new WG members
+Once acceptance has been acquired by group members, the following should take place to setup the new member:
+
+* Mentor should ping @jasnell, @gibfahn, @cjihrig or @mhdawson to add the new member to the repository.
+* New member should open a PR with his/her username added to the list in alphabetical order at: [security-wg/README.md at master · nodejs/security-wg · GitHub](https://github.com/nodejs/security-wg/blob/master/README.md)
+* New member should enable 2FA in GitHub
+* New member should join the Security WG slack medium and confirm his identity by providing necessary slack user details in the following private discussion: [Slack identity check](https://github.com/orgs/nodejs/teams/security-wg/discussions/3)
+
+## Setting-up for new Triage Team members
+Please follow:
+* New member should open a PR with his username added to [security-wg/third_party_vuln_process.md at master · nodejs/security-wg · GitHub](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md#the-triage-team), where the username list is ordered alphabetically.
+* New members should familiarize themselves with the 3rd party vulnerability triage process at: [security-wg/third_party_vuln_process.md at master · nodejs/security-wg · GitHub](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md)
+* New members should schedule a demo session with their mentor to review the HackerOne platform where the new member receives a walk-through of our processes.
```

Comment:
Is this the same session as the 30 minute touch-base call described in the "The Buddy Program" section or a separate one?

Reply:
yeah. but time length isn't a hard-limit inn advance.